helm/prometheus-rules/templates/platform/atlas/alerting-rules/tempo-mixins.rules.yml

```diff
@@ -0,0 +1,370 @@
+        - alert: TempoDistributorUnhealthy
+          annotations:
+            description: '{{`There are {{ printf "%f" $value }} unhealthy distributor(s).`}}'
+            runbook_url: https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/runbook.md#TempoDistributorUnhealthy
+          expr: |
+            max by (cluster_id, installation, pipeline, provider, namespace) (tempo_ring_members{cluster_type="management_cluster", state="Unhealthy", name="distributor"}) > 0
+          for: 15m
+          labels:
+            severity: page
+            area: platform
+            team: atlas
+            topic: observability
+            cancel_if_outside_working_hours: "true"
+        - alert: TempoIngesterUnhealthy
+          annotations:
+            description: '{{`There are {{ printf "%f" $value }} unhealthy ingester(s).`}}'
+            runbook_url: https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/runbook.md#TempoIngesterUnhealthy
+          expr: |
+            max by (cluster_id, installation, pipeline, provider, namespace) (tempo_ring_members{cluster_type="management_cluster", state="Unhealthy", name="ingester"}) > 0
+          for: 15m
+          labels:
+            severity: page
+            area: platform
+            team: atlas
+            topic: observability
+            cancel_if_outside_working_hours: "true"
+        - alert: TempoMetricsGeneratorUnhealthy
+          annotations:
+            description: '{{`There are {{ printf "%f" $value }} unhealthy metric-generator(s).`}}'
+            runbook_url: https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/runbook.md#TempoMetricsGeneratorUnhealthy
+          expr: |
+            max by (cluster_id, installation, pipeline, provider, namespace) (tempo_ring_members{cluster_type="management_cluster", state="Unhealthy", name="metrics-generator"}) > 0
+          for: 15m
+          labels:
+            severity: page
+            area: platform
+            team: atlas
+            topic: observability
+            cancel_if_outside_working_hours: "true"
+        - alert: TempoCompactionsFailing
+          annotations:
+            description: '{{`Greater than 2 compactions have failed in the past hour.`}}'
+            runbook_url: https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/runbook.md#TempoCompactionsFailing
+          expr: |
+            sum by (cluster_id, installation, pipeline, provider, namespace) (increase(tempodb_compaction_errors_total{cluster_type="management_cluster"}[1h])) > 2 and
+            sum by (cluster_id, installation, pipeline, provider, namespace) (increase(tempodb_compaction_errors_total{cluster_type="management_cluster"}[5m])) > 0
+          for: 1h
+          labels:
+            severity: page
+            area: platform
+            team: atlas
+            topic: observability
+            cancel_if_outside_working_hours: "true"
+        - alert: TempoIngesterFlushesUnhealthy
```

Comment:
Do we need both `TempoCompactionsFailing` and `TempoIngesterFlushesUnhealthy` alerts ?